analyzer/src/main/kotlin/managers/Gradle.kt

```diff
@@ -137,7 +138,9 @@ class Gradle(
             }
         }
 
-        val gradleConnector = GradleConnector.newConnector()
+        val gradleConnector = GradleConnector.newConnector().let {
+            if (gradleVersion != null) it.useGradleVersion(gradleVersion) else it
+        }
```

Comment:
Ah, right, I somehow though I'd need to use the return value, and as I wanted to avoid using a `var` instead of a `val`, I came up with this clumsy variant. Fixed.